### PR TITLE
[NO-TICKET] Wording change to v7 blog post

### PR DESCRIPTION
**Pull request by pwolfert**
## Summary

- Edit some wording in the v7 blog post to avoid giving the impression that teams should change the dropdown style

### CODE REVIEW

**Review by zarahzachz — CHANGES_REQUESTED** [review]

File: packages/docs/content/blog/release-7.0.mdx [hunk 1]
```diff
@@ -31,7 +31,7 @@ With the expansion of the design system themes to include a fourth theme (CMS.go
...
 
-In this update, we’ve resolved this issue by implementing the component as an ARIA combobox. A side benefit is that CMS brands can now customize the look and feel of the menu. The component has been redesigned from the ground up, and our Autocomplete component has also been updated to match the new designs. [Check out the Dropdown page to see it in action](/components/dropdown/).
+In this update, we’ve resolved this issue by implementing the component as an ARIA combobox. A side benefit is the menu can now styled to match CMS brands. The component has been redesigned from the ground up, and our Autocomplete component has also been updated to match the new designs. [Check out the Dropdown page to see it in action](/components/dropdown/).
```
[the remark this answers is not in the document]
Comment (pwolfert) [reply packages/docs/content/blog/release-7.0.mdx:34]: Good catch

[the remark this answers is not in the document]
Comment (brentschneider) [reply packages/docs/content/blog/release-7.0.mdx]: @pwolfert   
  
The previous suggestion you made I think drives home the point more succinctly.

> A side benefit is the component is now styled to match CMS brands.

_We could also condense it further_ ... 

> A side benefit is the component now matches CMS brands.

I don't believe either of these statements interferes with the next sentence as "redesigned from the ground up" is referring to engineering and design combined.  
  
  
cc @zarahzachz

[the remark this answers is not in the document]
Comment (pwolfert) [reply packages/docs/content/blog/release-7.0.mdx]: I think the detail I'm trying to convey in that sentence—that may or may not be important to communicate—is that it's the _menu_ style that couldn't be customized before, and now it _can_ be customized.

[the remark this answers is not in the document]
Comment (brentschneider) [reply packages/docs/content/blog/release-7.0.mdx]: @zarahzachz the thing that is making me twingy is "can now be" reads to me that it is open to changing the styling in stead of that it has been styled to match.

[the remark this answers is not in the document]
Comment (zarahzachz) [reply packages/docs/content/blog/release-7.0.mdx]: I think the dropdown itself was always customizable to work with themes; customizing the menu's the new thing. And emphasizing the menu's new ability to be customized plays well with the previous paragraph explaining the technical limitations of the native select menu. So with this menu we get a better UX _and_ flexible styling

[the remark this answers is not in the document]
Comment (pwolfert) [reply packages/docs/content/blog/release-7.0.mdx]: @brentschneider, would it alleviate your twinge to change the verb case to a hypothetical `could`? `could now be`? And the hypthetical then becomes concrete in our next sentence saying that _we did it_, and therefore you don't have to?

[the remark this answers is not in the document]
Comment (pwolfert) [reply packages/docs/content/blog/release-7.0.mdx]: Okay, I think this ballances succinctness, accuracy, and not suggesting any action for our readers:

> A side benefit is the menu is now styled to match CMS brands.

**Review by zarahzachz — APPROVED** [review]

**Pull request merged**